cmd/dep/graphviz.go

```diff
@@ -48,7 +48,7 @@ func (g graphviz) output() bytes.Buffer {
 	for _, dp := range g.ps {
 		for _, bsc := range dp.children {
 			for pr, hsh := range g.h {
-				if strings.HasPrefix(bsc, pr) {
+				if strings.HasPrefix(bsc, pr) && isPathPrefixOrEqual(pr, bsc) {
```

Comment:
I was imagining this as a standalone implementation - move the `strings.HasPrefix()` call inside `isPathPrefixOrEqual`, rather than requiring two calls out here.